### PR TITLE
Remove Runner ID from build information

### PR DESCRIPTION
**Pull request by daveverwer**
Runner ID is no longer very useful as it just either indicates Orka or Linux 1/2, so lets remove it.

### CODE REVIEW

**Review by finestructure — APPROVED** [review]

**Comment by finestructure** [comment]
Hold on a sec

File: Sources/App/Commands/Alerting.swift [hunk 1]
```diff
@@ -83,7 +83,6 @@ extension Alerting {
             self.updatedAt = build.updatedAt!
             self.builderVersion = build.builderVersion
             self.platform = build.platform
-            self.runnerId = build.runnerId
```
Comment (finestructure) [inline Sources/App/Commands/Alerting.swift:86]: We don't want to remove this from Alerting. The linux info can be helpful (and even the orka one could be if there's a stuck runner).

**Review by finestructure — CHANGES_REQUESTED** [review]
Need to keep the Alerting runnerId

File: Sources/App/Commands/Alerting.swift (hunk 1 above)
Comment (daveverwer) [reply Sources/App/Commands/Alerting.swift:86]: OK, so we need to leave the field and just remove it user-facing?

Comment (finestructure) [reply Sources/App/Commands/Alerting.swift:86]: Yes, exactly.

File: Sources/App/Commands/Alerting.swift [hunk 2]
```diff
@@ -123,7 +122,6 @@ extension Alerting {
             .field(\.$updatedAt)
             .field(\.$builderVersion)
             .field(\.$platform)
-            .field(\.$runnerId)
```
Comment (finestructure) [inline Sources/App/Commands/Alerting.swift:126]: Keep this as well.

Comment (finestructure) [reply Sources/App/Commands/Alerting.swift:126]: I.e. no changes in `Alerting.swift`, the rest should all be front-end only.

File: Sources/App/Migrations/078/RemoveBuildsRunnerId.swift [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Fluent
+
+struct RemoveBuildsRunnerId: AsyncMigration {
+    func prepare(on database: Database) async throws {
+        try await database.schema("builds")
+            .deleteField("runner_id")
```
[the remark this answers is not in the document]
Comment (finestructure) [reply Sources/App/Migrations/078/RemoveBuildsRunnerId.swift:20]: We definitely still want to report build ids.

File: Sources/App/Models/Build.swift [hunk 4]
```diff
@@ -71,9 +71,6 @@ final class Build: @unchecked Sendable, Model, Content {
     @Field(key: "platform")
     var platform: Platform
 
-    @Field(key: "runner_id")
-    var runnerId: String?
-
```
Comment (finestructure) [inline Sources/App/Models/Build.swift:76]: Need to keep this

File: Sources/App/Models/Build.swift [hunk 5]
```diff
@@ -101,7 +98,6 @@ final class Build: @unchecked Sendable, Model, Content {
         self.jobUrl = jobUrl
         self.logUrl = logUrl
         self.platform = platform
-        self.runnerId = runnerId
```
Comment (finestructure) [inline Sources/App/Models/Build.swift:104]: and this

File: Sources/App/Commands/Alerting.swift (hunk 2 above)
Comment (finestructure) [reply Sources/App/Commands/Alerting.swift:126]: No, there's more that we need to keep. Need time to look through.

**Comment by finestructure** [comment]
TBH, I think it'd be easier and safer to redo this and just limit the change to the build monitor page.

**Review by finestructure — APPROVED** [review]

**Pull request merged**